doc/source/installation/apt_repo.rst

```diff
@@ -146,51 +146,72 @@ This will install all the packages from the bundle. Another option is to downloa
+    :widths: 30 30
+    :header-rows: 1
 
-|Percona Server| stores the data files in :file:`/var/lib/mysql/` by default. You can find the configuration file that is used to manage |Percona Server| in :file:`/etc/mysql/my.cnf`.
+    * - Files
+      - Location
+    * - `mysqld` server
+      - :file:`/usr/sbin`
+    * - Configuration
+      - :file:`/etc/mysql/my.cnf`
+    * - Data directory
+      - :file:`/var/lib/mysql`
+    * - Logs
+      - :file:`/var/log/mysql`
 
 .. note::
 
-  *Debian* and *Ubuntu* installation doesn't automatically create a special ``debian-sys-maint`` user which can be used by the control scripts to control the |Percona Server| ``mysqld`` and ``mysqld_safe`` services like it was the case with previous |Percona Server| versions. If you still require this user you'll need to create it manually.
+  *Debian* and *Ubuntu* installation does not automatically create a special
+  ``debian-sys-maint`` user which can be used by the control scripts to control
+  the |Percona Server| ``mysqld`` and ``mysqld_safe`` services like it was the
+  case with previous |Percona Server| versions. If you still require this user you must create the user manually.
+
+Running |Percona Server|
+========================
+
+The following procedure runs the |Percona Server|:
 
 1. Starting the service
 
-   |Percona Server| is started automatically after it gets installed unless it encounters errors during the installation process. You can also manually start it by running:
+   |Percona Server| starts automatically after installation unless the server
+   encounters errors during the installation process. You can also manually
+   start it by running the following command:
 
    .. code-block:: bash
 
      $ sudo service mysql start
 
-2. Confirming that service is running
+2. Confirming the service is running
 
-   You can check the service status by running:
+   You can verify the service status by running the following command:
 
    .. code-block:: bash
 
      $ service mysql status
 
 3. Stopping the service
 
-   You can stop the service by running:
+   You can stop the service by running the following command:
 
    .. code-block:: bash
 
      $ sudo service mysql stop
 
 4. Restarting the service
 
-   You can restart the service by running:
+   You can restart the service by running the following command:
 
    .. code-block:: bash
 
      $ sudo service mysql restart
 
 .. note::
 
-  *Debian* 8.0 (jessie) and *Ubuntu* 15.04 (vivid) come with `systemd <http://freedesktop.org/wiki/Software/systemd/>`_ as the default system and service manager so you can invoke all the above commands with ``sytemctl`` instead of ``service``. Currently both are supported.
+  *Debian* 8.0 (jessie) and *Ubuntu* 15.04 (vivid) come with `systemd <http://freedesktop.org/wiki/Software/systemd/>`_ as the default system and service manager so you can invoke all the above commands with ``sytemctl`` instead of ``service``. Currently, both are supported.
```

Comment:
Maybe put Ubuntu 16.04 (Xenial) instead Vivid since Vivid is no longer supported and was not LTS release anyway.

Reply:
updated to 16.04